### ISSUE TITLE
Dynamic Shipping Tax Class Calculation

### ISSUE THREAD

**Issue by therouv** [firegento/firegento-magesetup2#22]
Calculate shipping tax class depending on products in cart:
- Use highest product tax class as shipping tax class
- Or use the tax rate of products that make up the biggest amount

**Comment by m4c3** [comment]
Hi,
when can we expect this feature?
thanks Matthias

**Comment by aqdennis** [comment]
Indeed. It was available in MageSetup for Magento 1.9, but cannot find it in MageSetup 2....

**Comment by frostblogNet** [comment]
We will close this issue from 2017. Feel free to reopen if it is relevant.

Thanks

**Comment by sprankhub** [comment]
If I understand correctly, https://github.com/firegento/firegento-magesetup2/commit/525797794a59f4bad13653ecb20f124c4fd45c32 has not been merged, so this is not implemented yet. IMHO, it makes sense to add that, so I think we should keep that issue open.

**Comment by BorisovskiP** [comment]
I will be working on this

**Comment by nxb303** [comment]
Thank you for your work on this module.
What's the status on this?

**Comment by sprankhub** [comment]
This feature is available as a third party extension: https://github.com/JaJuMa-GmbH/magento-2-dynamic-shipping-tax
